### PR TITLE
Add timezone-buddy extension

### PR DESCRIPTION
**Pull request by andrebreia**
## Description

This is a small extension that allows creating a list of friends in different timezones and seeing their current time.

## Screencast

https://github.com/raycast/extensions/assets/17434432/40654ce2-9849-42d9-816b-6a3e97e9caf1

## Checklist

- [x] I read the [extension guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store)
- [x] I read the [documentation about publishing](https://developers.raycast.com/basics/publish-an-extension)
- [x] I ran `npm run build` and [tested this distribution build in Raycast](https://developers.raycast.com/basics/prepare-an-extension-for-store#metadata-and-configuration)
- [x] I checked that files in the `assets` folder are used by the extension itself
- [x] I checked that assets used by the `README` are placed outside of the `metadata` folder

### CODE REVIEW

File: extensions/timezone-buddy/src/timezone-buddy.tsx [hunk 1]
```diff
@@ -0,0 +1,298 @@
...
+      target={<CreateBuddyForm onCreate={props.onCreate} />}
+    />
+  );
+}
+
+function DeleteBuddyAction(props: { onDelete: () => void }) {
+  return (
+    <Action
+      icon={Icon.Trash}
+      title="Delete Buddy"
```
Comment (pernielsentikaer) [inline extensions/timezone-buddy/src/timezone-buddy.tsx:125]:
```suggestion
      title="Delete Buddy"
      style={Action.Style.Destructive}
```

File: extensions/timezone-buddy/src/timezone-buddy.tsx [hunk 2]
```diff
@@ -0,0 +1,298 @@
...
+        : getAvatarIcon(values.name),
+    });
+    pop();
+  }
+
+  return (
+    <Form
+      actions={
+        <ActionPanel>
+          <Action.SubmitForm title="Add Buddy" onSubmit={handleSubmit} />
```
Comment (pernielsentikaer) [inline extensions/timezone-buddy/src/timezone-buddy.tsx:84]:
```suggestion
          <Action.SubmitForm title="Add Buddy" icon={Icon.AddPerson} onSubmit={handleSubmit} />
```

File: extensions/timezone-buddy/src/timezone-buddy.tsx [hunk 3]
```diff
@@ -0,0 +1,298 @@
...
+      </Form.Dropdown>
+    </Form>
+  );
+}
+
+function CreateBuddyAction(props: { onCreate: (buddy: TimezoneBuddy) => void }) {
+  return (
+    <Action.Push
+      icon={Icon.Pencil}
+      title="Add Buddy"
```
Comment (pernielsentikaer) [inline extensions/timezone-buddy/src/timezone-buddy.tsx:114]:
```suggestion
      title="Add Buddy"
      icon={Icon.AddPerson}
```

**Comment by andrebreia** [comment]
Thanks for the feedback @pernielsentikaer. I somehow missed the GitHub notification about these comments, I'll review them this week 👍

**Comment by pernielsentikaer** [comment]
Awesome @andrebreia 🔥

**Comment by andrebreia** [comment]
@pernielsentikaer I've updated the extension based on your feedback!


https://github.com/raycast/extensions/assets/17434432/6307a5f8-60e0-4216-acf1-0539233f044b

**Comment by pernielsentikaer** [comment]
Awesome updates, I keep seeing NaN in every person I add, is that a regression?

timezone-buddy 2024-04-03 at 17 25 45

**Comment by Pe8er** [comment]
@andrebreia Sorry for off-topic question: where did you get your wallpaper from? Can you share it please?

**Comment by pernielsentikaer** [comment]
Hint: https://www.raycast.com/wallpapers @Pe8er 😉

**Comment by andrebreia** [comment]
@pernielsentikaer It should be fixed - it was an issue with the locale when calculating the offset 👍 


@Pe8er I'm a big fan of Raycast wallpapers!

**Comment by pernielsentikaer** [comment]
Should we consider adding so it's possible to sort buddies by moving them up and down? That might be something we should look into in the first release if data needs to change.

Let me know what you think about that 🙂

**Comment by andrebreia** [comment]
@pernielsentikaer Good idea! I'll try to get it done later in the week.

**Comment by andrebreia** [comment]
@pernielsentikaer Sorry, it's been busy for the last couple of weeks.

I'm almost done with the moving up/down feature. I'll get it ready for PR next Friday.

**Comment by pernielsentikaer** [comment]
All good, no rush at all, just wanted to check if I could help you 🙂

**Review by pernielsentikaer — APPROVED** [review]
Hi 👋 

Looks good to me, approved 🔥

**Pull request merged**